sentry/connection_config.go

```diff
@@ -47,11 +51,31 @@ func getClient(ctx context.Context, d *plugin.QueryData) (*sentry.Client, error)
 	sentryConfig := GetConfig(d.Connection)
 
 	authToken := os.Getenv("SENTRY_AUTH_TOKEN")
+	baseUrl := os.Getenv("SENTRY_URL")
+
+	if sentryConfig.BaseUrl != nil {
+		baseUrl= *sentryConfig.BaseUrl
+	}
 
 	if sentryConfig.AuthToken != nil {
 		authToken = *sentryConfig.AuthToken
 	}
 
+	if baseUrl != "" {
```

Comment:
Are you able to shorten this code? Getting the token, creating the client, etc., all look to be shared with some of the code below, so consolidation would be better if possible.